job-roles/product-manager.md

```diff
@@ -0,0 +1,41 @@
+# Product Manager
+
+{{#include intro.md}}
+
+## Role description
+As a MayaData Product Owner you will have the exciting opportunity to help drive the growth and shape the future of Kubera and OpenEBS through building a vibrant contributor community. In this highly technical role, you will be responsible for specific MayaData subsystems and work with our Engineering and Product teams responsible for building and delivering them to market. You will be responsible for planning and delivering product content for new releases and launches of our offerings, prioritizing and communicating product requirements to guide development, supporting the field team in advanced sales opportunities, performing competitive reviews and analysis, and working with customers and partners. You will actively contribute to the overall company strategy and work closely with Engineering, Marketing, Support, Sales, and Field teams to ensure our offerings' success. You are also expected to engage with open source communities that support our container initiatives, including Kubernetes, other Cloud Native Computing Foundation (CNCF) projects, CSI, Velero, and more. As a Product Owner, you need to have great communication, teamwork, and persuasion skills. This is a great opportunity to work on a fast-growing offering alongside some of the brightest minds in open source. 
```

Comment:
This role is titled "Product Manager" but  the first sentence of the Role Description refers to "Product Owner".  Is this intentional?